### PR TITLE
[Enhancement]: Added rest of the CSS grid properties

### PR DESCRIPTION
**Pull request by Paul-Annay**
## What changes does this PR introduce?

In this PR the [styles.go](https://github.com/chasefleming/elem-go/blob/main/styles/styles.go) file is updated by adding the  grid-properties (mentioned below) following the [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/grid). 
```CSS
grid-auto-columns,
grid-auto-rows,
grid-column-gap,
grid-gap,
grid-row-gap,
grid-template-areas,
grid-template-columns,
grid-template-rows,
```

## Fixes:
Issue #14

### CODE REVIEW

File: styles/styles.go [hunk 1]
```diff
@@ -2,31 +2,40 @@ package styles
...
+	AlignContent        = "align-content"
+	FlexWrap            = "flex-wrap"
+	Grid                = "grid"
+	GridArea            = "grid-area"
+	GridAutoColumns     = "grid-auto-columns"
+	GridAutoFlow        = "grid-auto-flow"
+	GridAutoRows        = "grid-auto-rows"
+	GridColumn          = "grid-column"
+	GridColumnEnd       = "grid-column-end"
+	GridColumnGap       = "grid-column-gap"
```
Comment (chasefleming) [inline styles/styles.go:28]: Deprecated in favor of `column-gap`. Let's just change it to `ColumnGap` and `column-gap`

File: styles/styles.go [hunk 2]
```diff
@@ -2,31 +2,40 @@ package styles
...
+	GridColumnStart     = "grid-column-start"
+	GridRow             = "grid-row"
+	GridGap             = "grid-gap"
```
Comment (chasefleming) [inline styles/styles.go:31]: Deprecated in favor of `gap`. Let's just change it to `Gap` and `gap`.

File: styles/styles.go [hunk 3]
```diff
@@ -2,31 +2,40 @@ package styles
...
+	GridRowEnd          = "grid-row-end"
+	GridRowGap          = "grid-row-gap"
```
Comment (chasefleming) [inline styles/styles.go:33]: Deprecated in favor of `row-gap`. Let's just change it to `RowGap` and `row-gap`.

**Review by chasefleming — CHANGES_REQUESTED** [review]
Awesome, thank you! Looks great. I just left a couple notes on a few that were deprecated for different syntax.

**Comment by Paul-Annay** [comment]
> Awesome, thank you! Looks great. I just left a couple notes on a few that were deprecated for different syntax.

@chasefleming  thank you for your prompt review, I have made the suggested changes. Feel free to send more feedback if necessary.

**Review by chasefleming — APPROVED** [review]
Looks good, thanks @Paul-Annay !

**Pull request merged**